docs/dev/hacking-guide.md

```diff
@@ -0,0 +1,233 @@
+# Machine API - Hacking guide
+
+## Important
+This guide assumes that the reader is familiar with Machine API and it’s resources(Machines, MachineSets, etc. https://docs.openshift.com/container-platform/4.5/machine_management/creating_machinesets/creating-machineset-aws.html). 
+
+Frequently asked questions can be found here - https://github.com/openshift/machine-api-operator/blob/master/FAQ.md
```

Comment:
I thought this will become a clickable once merged.

Reply:
Ahh yeah, they do become clickable, just not pretty with the exposed URLs. I don't have a strong opinion, ie not blocking, but I tend to put these as `[some text](url)` style links so that the text can be part of the sentence and the URL is discreet